spec/pubid_iso/identifier_spec.rb

```diff
@@ -0,0 +1,19 @@
+RSpec.describe PubidIso::Identifier do
+  subject { described_class.parse(pubid) }
+
+  context "ISO 4" do
+    let(:pubid) { "ISO 4" }
+
+    it "returns correct URN" do
+      expect(subject.urn.to_s).to eq("urn:iso:std:iso:4")
+    end
+  end
+
+  context "ISO/IEC FDIS 7816-6" do
+    let(:pubid) { "ISO/IEC FDIS 7816-6" }
+
+    it "returns correct URN" do
+      expect(subject.urn.to_s).to eq("urn:iso:std:iso-iec:7816:-6:stage-50.00")
+    end
+  end
```

Comment:
@ronaldtse what do you think about this? Is it correct?

Reply:
I think so. Technically, maybe this should be stage-50.60 because if I see this identifier I would think is a published document.

Reply:
Actually 50.00 is correct. Ignore my previous comment.